tensorbay/label/label_mask.py

```diff
@@ -355,7 +355,20 @@ def get_callback_body(self) -> Dict[str, Any]:
         return body
 
 
-class RemoteSemanticMask(SemanticMaskBase, RemoteFileMixin):
+class RemoteMaskErrorMixin:  # pylint: disable=too-few-public-methods
+    """RemoteMaskErrorMixin provid more accurate error information when calling ``dumps()``."""
+
+    def get_callback_body(self) -> None:
+        """``dumps`` function is not support for remote mask label.
+
+        Raises:
+            AttributeError: when calling the ``dumps()``.
+
+        """
+        raise AttributeError(f"'{self.__class__.__name__}' object has no function 'dumps()'")
```

Comment:
The error message is weird because the function name is `get_callback_body` but the message is "no function `dumps`"?